views/_instructions.html

```diff
@@ -1,25 +1,21 @@
 {% macro r(dataset) -%}
 <p>In order to use Data Package in R follow instructions below:</p>
 <div class="highlight">
-<pre>
-<span></span>install.packages<span class="p">(</span><span class="s">"devtools"</span><span class="p">)</span>
-<span class="kn">library</span><span class="p">(</span>devtools<span class="p">)</span>
-install_github<span class="p">(</span><span class="s">"hadley/readr"</span><span class="p">)</span>
-install_github<span class="p">(</span><span class="s">"ropenscilabs/jsonvalidate"</span><span class="p">)</span>
-install_github<span class="p">(</span><span class="s">"ropenscilabs/datapkg"</span><span class="p">)</span>
+<pre class="hljs"><code>install.packages(<span class="hljs-string">"rjson"</span>)
+<span class="hljs-keyword">library</span>(<span class="hljs-string">"rjson"</span>)
 
-<span class="c1">#Load client</span>
-<span class="kn">library</span><span class="p">(</span>datapkg<span class="p">)</span>
+json_file &lt;- <span class="hljs-string">"https://pkgstore.datahub.io/core/finance-vix/latest/datapackage.json"</span>
```

Comment:
This is the key point and it should not be hard coded. What happens if you go to another dataset? There will be the same path which is wrong. Use variable here, see JS instrucitons example.

Can you also fix the same thing for Python instructions?